test/runnable/testargtypes.d

```diff
@@ -104,6 +104,13 @@ version (X86_64)
         struct S9 { int a,b,c; }
         chkArgTypes!(S9, long, int)();
 
+        struct S10 { int[3] a; }
+        chkArgTypes!(S10, long, int)();
+
+        chkArgTypes!(int[3], long, int)();
+
+        chkArgTypes!(S9[1], long, int)();
```

Comment:
Are you checking `S9` or `S10` here?  Tests against `S9` should be grouped together to prevent confusion.

Reply:
this was fixed.